src/rsa/public/exponent.rs

```diff
@@ -77,6 +78,19 @@ impl Exponent {
 
         Ok(Self(value))
     }
+
+    /// The big-endian encoding of the exponent.
+    ///
+    /// There are no leading zeros.
```

Comment:
Nit: This may be better worded as `All leading zeros have been stripped from the value.`

Reply:
I thought about that too, but there's no reason to imply that there were any leading zeros in the first place, as that's an implementation detail.

Reply:
I see. If that's an implementation detail then the current comment is fine.